operator/v1/types_ingress.go

```diff
@@ -1037,6 +1037,16 @@ type SyslogLoggingDestinationParameters struct {
 // ContainerLoggingDestinationParameters describes parameters for the Container
 // logging destination type.
 type ContainerLoggingDestinationParameters struct {
+	// maxLength is the maximum length of the syslog message
+	//
+	// If this field is empty, the maxLength is set to "1024".
```

Comment:
Please mention the minimum and maximum values.  (The values specified by `kubebuilder:validation:Maximum` and `kubebuilder:validation:Minimum` don't show up in the `oc explain` output.)